src/Gaffer/Context.cpp

```diff
@@ -304,15 +307,15 @@ IECore::MurmurHash Context::hash() const
 		// Using a hardcoded comparison of the first three characters because
 		// it's quicker than `string::compare( 0, 3, "ui:" )`.
 		const std::string &name = it->first.string();
-		if(	name.size() > 2 && name[0] == 'u' && name[1] == 'i' && name[2] == ':' )
+		if( name.size() > 2 && name[0] == 'u' && name[1] == 'i' && name[2] == ':' )
```

Comment:
Oh, that's totally fair.  I was eager to get it out of `set()` because V1 required this logic to match in `set`, `remove` and `removeMatching`, which felt horrifically fragile.  But now that we don't care about removal, putting it in `set` is actually fairly nice.

Reply:
Put it in the new updateHash function on Storage, which feels pretty nice now.

Reply:
Fixed in 7e95c0f